mapper/complementaryContentMapper.go

```diff
@@ -74,19 +82,19 @@ func buildCCAlternativeTitles(promoTitle string) *model.AlternativeTitles {
 	return &model.AlternativeTitles{PromotionalTitle: promoTitle}
 }
 
-func buildCCAlternativeImages(fileRef string) *model.AlternativeImages {
+func (ccm *ComplementaryContentCPHMapper) buildCCAlternativeImages(fileRef string) *model.AlternativeImages {
 	if fileRef == "" {
 		return nil
 	}
 	imageUUID := extractImageUUID(fileRef)
-	return &model.AlternativeImages{PromotionalImage: imageUUID}
+	return &model.AlternativeImages{PromotionalImage: &model.PromotionalImage{Id: fmt.Sprintf(ccm.apiHostFormat, imageUUID)}}
 }
 
 func extractImageUUID(fileRef string) string {
 	return strings.Split(fileRef, "uuid=")[1]
 }
 
-func buildCCAlternativeStandfirsts(promoStandfirst string) *model.AlternativeStandfirsts {
+func (ccm *ComplementaryContentCPHMapper) buildCCAlternativeStandfirsts(promoStandfirst string) *model.AlternativeStandfirsts {
```

Comment:
What is the purpose of this function having a ComplementaryContentCPHMapper receiver?

Reply:
Same